dockstore-webservice/src/main/java/io/dockstore/webservice/core/CloudInstance.java

```diff
@@ -0,0 +1,121 @@
+package io.dockstore.webservice.core;
+
+import java.io.Serializable;
+import java.util.Set;
+
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(value = "CloudInstance", description = "Instances that launch-with cloud partners have")
+@Entity
+@Table(name = "cloud_instance")
+@NamedQueries({
+        @NamedQuery(name = "io.dockstore.webservice.core.CloudInstance.findAllWithoutUser", query = "SELECT ci from CloudInstance ci where user_id is null")
+})
+public class CloudInstance implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", unique = true, nullable = false)
+    @ApiModelProperty(value = "The ID used to update CloudInstances", accessMode = ApiModelProperty.AccessMode.READ_ONLY)
+    private long id;
+
+    // TODO: This needs to be an enum
+    // @Column(name = "partner", nullable = false, columnDefinition = "enum('GALAXY', 'TERRA', 'DNA_STACK', 'DNA_NEXUS', 'CGC', 'NHLBI_BIODATA_CATALYST', 'ANVIL')") did not work
+    @Column(name = "partner", nullable = false)
+    @ApiModelProperty(value = "Name of the partner")
+    @Enumerated(EnumType.STRING)
+    private Partner partner;
+
+    @Column(name = "url", nullable = false)
+    @ApiModelProperty(value = "The URL of the launch-with partner's private cloud instance")
+    private String url;
+
+    @Column(name = "supports_http_imports")
+    @ApiModelProperty(value = "Whether the CloudInstance supports http imports or not")
```

Comment:
`https`, to match the name of the property/column

Reply:
can you elaborate?

Reply:
Never mind, brain fart.